packages/hardhat-vyper/src/parser.ts

```diff
@@ -31,6 +33,16 @@ export class Parser {
     return result;
   }
 
+  private _validateTestModeNotUsed(fileContent: string, absolutePath: string) {
+    if (fileContent.includes('#@ if mode == "test":')) {
+      throw new Error(
+        `We found a test directive in the file at path ${absolutePath}.` +
+          ` Test directives are a Brownie feature not supported by Hardhat.` +
+          ` Learn more at https://hardhat.org/hardhat-runner/plugins/nomiclabs-hardhat-vyper#test-directives.`
```

Comment:
Let's remove the dot just in case, maybe it messes with the link in some terminals

```suggestion
          ` Learn more at https://hardhat.org/hardhat-runner/plugins/nomiclabs-hardhat-vyper#test-directives`
```